.travis.yml

```diff
@@ -8,6 +8,7 @@ go_import_path: github.com/Colstuwjx/awx-go
 
 install:
   - go get -u github.com/alecthomas/gometalinter
+  - go get -u github.com/kylelemons/godebug/pretty
```

Comment:
Should we add this dependency inside travis, rather than adding into vendor?
It may caused `go test -v *.go` such way broken.

Reply:
Hi Luca, can you please use govendor to manage dependencies?
Please, find below a quick and dirty guide. It will save also a couple of build minutes in complex project and make your dependencies management more robust.
1. install govendor
go get -u github.com/kardianos/govendor

2. initialize the vendor (create the vendor folder and the configuration file
govendor init

3. Add GOPATH files and missed packages to the vendor folder
govendor add +external +missing

4. Change the .travis.yml file just to install govendor
install:
  - go get -u github.com/kardianos/govendor

script:
  - make vendor-status

5. Change Makefile adding a rule to lists any packages missing, out-of-date, or modified locally.
i.e.
vendor-status:
  govendor status